lib/spatial_features/download.rb

```diff
@@ -4,18 +4,28 @@ module SpatialFeatures
   module Download
     # file can be a url, path, or file, any of which can return be a zipped archive
     def self.read(file, unzip: nil, **unzip_options)
-      file = open(file, unzip: unzip, **unzip_options)
+      file = Download.open_each(file, unzip: unzip, **unzip_options).first
       path = ::File.path(file)
       return ::File.read(path)
     end
 
-    def self.open(file, unzip: nil, **unzip_options)
+    # file can be a url, path, or file, any of which can return be a zipped archive
+    def self.open(file)
```

Comment:
This changes the return type from a single `File` to `[File, File, ....]` - these methods seem like they should be marked private.  We may want to do a semver bump as well.